__tests__/lamp.test.js

```diff
@@ -1,12 +1,22 @@
 const Lamp = require('../src/lamp.js');
 
+let bulb;
+beforeEach(() => {
+    bulb = {
+        shelfLife: 5,
+        isWorking: true,
+        //test-doubles: dummy here inside of a stub
```

Comment:
This isnt really needed here, I dont know if it was a requirement of the task to include comments highlighting what test methods were used bu if it were, it could read
```suggestion
        // Dummy here inside of a stub
```
Best to use sentence case with a space after the //